.idea/php.xml

```diff
@@ -219,13 +219,15 @@
       </server>
     </servers>
   </component>
-  <component name="PhpProjectSharedConfiguration" php_language_level="7.3" />
+  <component name="PhpProjectSharedConfiguration" php_language_level="8.0">
```

Comment:
This looks OK, confirmed by Chris who is using PHPStorm more than me

Reply:
Maybe I'll just revert the changes to the .idea directory altogether.